docs/rewards/proof-of-location.mdx

```diff
@@ -5,13 +5,17 @@ sidebar_label: Proof of Location  (PoL)
 
 The **Proof-of-Location** algorithm evaluates location data associated with the station, their accuracy, and consistency. It generates a score which is a confidence indicator about the station’s location. WeatherXM places great importance on station continuity, and discourage relocations.
 
-### Proof-of-Location (PoL): 
+## Proof-of-Location (PoL) 
 PoL is a rational number in the range [0,1].
 A station’s PoL must be greater than or equal to PoL_Threshold to be considered for inclusion in WeatherXM's **Rewarding Mechanism**.
-- PoL: is calculated based on the station’s location, and the station’s PoL history. 
-- PoL: is a dynamic value, and is recalculated every time a station’s location is updated.
+PoL: is a dynamic value, and is recalculated every time a station’s location is updated.
```

Comment:
Let's replace "PoL: is" with "PoL is" due to the fact that it's in the same line as the previous sentence.